lib/crypto.js

```diff
@@ -647,9 +647,21 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
-
-exports.rng = exports.prng = randomBytes;
+exports.randomBytes = util._deprecateSync(randomBytes,
+    '`crypto.randomBytes(size)` is deprecated. ' +
+    'Please use `crypto.randomBytesSync(size)` instead.');
+exports.randomBytesSync = randomBytes;
+exports.pseudoRandomBytes = util.deprecate(randomBytes,
+    '`crypto.pseudoRandomBytes()` is deprecated. ' +
+    'Please use `crypto.randomBytes()` instead.');
```

Comment:
okay i'm not 100% sure what changes have been made and what's the best wording. feel free to rewrite in a separate PR if that's easier - i don't care about receiving credit. i just want to see this deprecation merged!